.circleci/config.yml

```diff
@@ -180,7 +180,10 @@ jobs:
             docker login -u $DOCKER_USER -p $DOCKER_PASS
             docker build --pull -t addons-frontend .
       - run: docker images
-      - run: docker run --rm addons-frontend node --version
+      - run:
+          name: Build the app and check bundlesize from container
+          command: |
+            docker run --rm addons-frontend yarn build-ci
```

Comment:
This is going to fail the deploy if bundlesize isn't happy, which is probably the last thing we want during a hotfix deploy (or any other kind of deploy in fact).

Reply:
I know, but I think this is worth it. This shouldn't suddenly start failing for the tag, because we would already have run that check outside a container for PRs, and on `master` commits as well. So we should have other occasions to see that fail before it starts blocking a deploy, unless we're blindly yolo-ing a change, and even if we do, it shouldn't be something big enough that it could cause the bundle size to grow beyond the threshold.

I don't want to start depending on `docker` for PRs, because of the possibility of running untrusted code that could steal docker credentials that we need to build because of the dockerhub limits around pulling images/layers.